hack/test-e2e-local.sh

```diff
@@ -19,6 +19,12 @@ ginkgo_flags=
 if [ -n "${CI:-}" -a -n "${ARTIFACTS:-}" ] ; then
   mkdir -p "$ARTIFACTS"
   ginkgo_flags="--output-dir=$ARTIFACTS --junit-report=junit.xml"
+  
+  printf "\n127.0.0.1 api.e2e-default.local.external.local.gardener.cloud\n127.0.0.1 api.e2e-default.local.internal.local.gardener.cloud" >> /etc/hosts
```

Comment:
afaik there is nothing special about `/etc/hosts` file in prow job pods, so it should.

Nit: it might be good to close the statement with an `\n` for the case someone else will add lines to the file in the future.